3_navigation.ipynb

```diff
@@ -83,7 +155,29 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Now, we specify the goal of autonomous movement from RViz. Click \"2D Nav Goal\" on the toolbar at the top of RViz and click an unobstructed location in the room:\n",
```

Comment:
"Now, we specify the autonomous movement goal using RViz. Click \"2D Nav Goal\" in the toolbar at the top of RViz and click an unobstructed location in the room:\n",